docs/usage.rst

```diff
@@ -33,14 +33,26 @@ the box:
 
 Template tags
 ~~~~~~~~~~~~~
-Two template tags are included
-:meth:`~user_sessions.templatetags.user_sessions.device` and
-:meth:`~user_sessions.templatetags.user_sessions.location`. These can be used
-for respectively humanizing the user agent string and showing an approximate
-location of the IP address::
+Four template tags are included.
+
+The first three are
+:meth:`~user_sessions.templatetags.user_sessions.browser`, which can be used
+to get just the browser from a session, and
+:meth:`~user_sessions.templatetags.user_sessions.platform`, which can be used
+to get just the hardware or operating system from a session, and
+:meth:`~user_sessions.templatetags.user_sessions.device`, which can be used to
+get both from a sesesion::
```

Comment:
These might be better presented as a list instead of mentioning them in a single paragraph.

Reply:
Good idea! Fixed.